api/lanes.py

```diff
@@ -571,6 +572,7 @@ def url_arguments(self):
             languages=self.language_key,
             audiences=self.audience_key
         )
+        kwargs[Facets.ORDER_FACET_GROUP_NAME] = Facets.ORDER_SERIES_POSITION
```

Comment:
I think I can take this out now, since I made series position the default facet.